python/ct/client/log_client.py

```diff
@@ -209,18 +213,29 @@ def __str__(self):
     def get_response(self, uri, params=None):
         """Get an HTTP response for a GET request."""
         try:
-            return requests.get(uri, params=params, timeout=self._timeout,
-                                verify=self._ca_bundle)
-        except requests.exceptions.RequestException as e:
+            return RequestHandler._build_requests_style_response(
+                self._http.request(RequestHandler._uri_with_params(uri, params)))
+        except Exception as e:
```

Comment:
Isn't this except clause a bit broad? Does httplib2 not specify the type of exceptions it'll raise?